src/components/carousel/panels/subcomponents/StateNote.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+export const StateNote = styled.span`
+color: #eee;
```

Comment:
lost some formatting here. obviously not a big deal, but perhaps we could consider adding a lint:fix script later to make this a non-issue in the future